### PR TITLE
Fix: Transparency regarding what is meant by 'empt' classrooms

### LINKED ISSUE #47: Transparency regarding what is meant by "empty" classrooms
**Is your feature request related to a problem? Please describe.**
Users think that the classrooms listed as empty are meant to be available as "free to use". This is not always the case. Classrooms listed as empty only mean that those rooms are supposed to be free of any classes according to the timetable at ERP. Many a times professors change the classroom which is not reflected in ERP. Also suppose a classroom has no classes in a particular day, it is possible that the guards might decide to not unlock the room at all making it inaccessible (very often the case with Vikramshila classrooms).

**Describe the solution you'd like**
Show a single line info banner on top of the website with a text like "The empty classes listed below might not be accurate in real life." There will also be a learn more button which will redirect to a page which clearly explains the various situations described above.

**Describe alternatives you've considered**
People will understand - Does not work

### PR DESCRIPTION
**Pull request by Swarnendu0123**
# Description

added Learn More component to clarify The empty classes listed below might not be accurate in real life.

Fixes #47

## Type of change
Screenshot (32)
Screenshot (33)

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/src/LearnMoreContent.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const LearnMoreContent = () => {
+  return (
+    <div>
+      <h3>Understanding Classroom Availability: Rules and Guidelines</h3>
+      <p>
+        1. Timetable Accuracy: Empty classrooms are based on the official
+        timetable provided by the institution's ERP system. Professors might
```
Comment (chirag-ghosh) [inline frontend/src/LearnMoreContent.js:9]: This feels like it is written by ChatGPT

**Review by chirag-ghosh — CHANGES_REQUESTED** [review]
Please add screenshots showing changes in the page.

File: frontend/src/LearnMoreContent.js (hunk 1 above)
Comment (Swarnendu0123) [reply frontend/src/LearnMoreContent.js]: Yes! I have taken help from ChatGPT for the content. But I have checked multiple times, the content generated by ChatGPT is good to go. Kindly, check from you end also.

**Comment by proffapt** [comment]
@chirag-ghosh review this once please.

**Comment by chirag-ghosh** [comment]
This needs to be rebased since there are significant changes in the frontend code. @Swarnendu0123 can you please fix the conflicts?

**Comment by Swarnendu0123** [comment]
This file is deleted from `master`. Nad the project also converted to typescript. When I created this PR, it was in js. Closing it for now.

**Pull request closed without merging**